frontend/views/pages/GroupSettings.vue

```diff
@@ -1,9 +1,9 @@
 <template lang="pug">
 page(pageTestName='dashboard' pageTestHeaderName='groupName' v-if='groupSettings.groupName')
   template(#title='') Group Settings
+  template(#description='') Changes to these settings will be visible to all group members
```

Comment:
Should be `i18n(tag='template'` I think (or using `{{ L(' ... ') }}` if that doesn't work for some reason

Reply:
No, the translation is done in the template already